pkg/providers/aws/provider_postgres.go

```diff
@@ -59,6 +59,8 @@ const (
 	defaultPostgresUserKey               = "user"
 	defaultStorageEncrypted              = true
 	postgresProviderName                 = "aws-rds"
+	existingCert                         = "rds-ca-2019"
```

Comment:
should this change be here? It seems this is from Austin PR? Possibly a rebase?

Reply:
Thank you for comment. Rebase done, but it was complex, and this remains, But changes from Austin's PR(s) were added as separate commit's and it can be found in the list of commit's.

Reply:
But we don't need/want that commit, let's remove it. Happy to help with rebase if you are stuck

Reply:
Done. removed/reverted - https://github.com/integr8ly/cloud-resource-operator/pull/798/commits/bd3644333a0770a7ca360e0e049ba3000bf714bc